migrations/013.do.published-as.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE retrievals ADD COLUMN published_as VARCHAR(64);
```

Comment:
Can we rely on CIDs to be at most 64 characters long?

According to https://maximorlov.com/char-varchar-text-postgresql/, which quotes from the official Postgres docs, there is no performance benefit in using `char` or `varchar` instead of `text`.

> There is no performance difference among these three types, apart from increased storage space when using the blank-padded type, and a few extra CPU cycles to check the length when storing into a length-constrained column. While character(n) has performance advantages in some other database systems, there is no such advantage in PostgreSQL; in fact character(n) is usually the slowest of the three because of its additional storage costs. In most situations text or character varying should be used instead.

```suggestion
ALTER TABLE retrievals ADD COLUMN published_as TEXT;
```

Reply:
I'm not very certain how large a CID can be. This is definitely safer, so committed it 👍 

Space is a consideration here, since our spark db instance sometimes runs out of memory. I don't think we have to be this precise though